ui/app/AppLayouts/WalletV2/components/AddAccountPopup.qml

```diff
@@ -0,0 +1,211 @@
+                        }
+                    }
+                }
+            }
+        }
+
+        Separator {
+            id: separator
+            width: keyOrSeedPhraseInput.width
+            anchors.top: keyOrSeedPhraseInput.bottom
+            anchors.topMargin: popup.marginBetweenInputs
+        }
+
+        Row {
+            id: accountNameInputRow
+            anchors.left: parent.left
+            anchors.leftMargin: 8
+            anchors.right: parent.right
+            anchors.rightMargin: 10
+            anchors.top: separator.bottom
+            height: 82
+            spacing: 10
+            Item {
+                implicitWidth: 434
+                height: parent.height
+                StatusInput {
+                    id: accountNameInput
+                    anchors.fill: parent
+                    input.implicitHeight: 56
+                    input.placeholderText: qsTrId("enter-an-account-name...")
+                    label: qsTrId("account-name")
+                    validators: [StatusMinLengthValidator { minLength: 1 }]
+                    onTextChanged: {
+                        errorMessage = (accountNameInput.text === "") ?
+                                    qsTrId("you-need-to-enter-an-account-name") : ""
+                    }
+                }
+            }
+            Item {
+                //emoji placeholder
+                width: 80
+                height: parent.height
+                anchors.top: parent.top
+                anchors.topMargin: 11
+                StyledText {
+                    id: inputLabel
+                    text: "Emoji"
+                    font.weight: Font.Medium
+                    font.pixelSize: 13
+                    color: Style.current.textColor
+                }
+                Rectangle {
+                    width: parent.width
+                    height: 56
+                    anchors.top: inputLabel.bottom
+                    anchors.topMargin: 7
+                    radius: 10
+                    color: "pink"
+                    opacity: 0.6
+                }
+            }
+        }
+
+        StatusInput {
+            id: passwordInput
+            anchors.left: parent.left
+            anchors.leftMargin: 8
+            anchors.right: parent.right
+            anchors.rightMargin: 10
+            implicitHeight: 82
+            input.implicitHeight: 56
+            anchors.top: accountNameInputRow.bottom
+            anchors.topMargin: popup.marginBetweenInputs
+            input.placeholderText: qsTr("Give the account a secure password")
+            label: qsTr("Password")
+            input.echoMode: TextInput.Password
+            validators: [StatusMinLengthValidator { minLength: 6 }]
+            onTextChanged: {
+                errorMessage = (passwordInput.text === "") ?
+                            qsTrId("you-need-to-enter-a-password") :
+                            qsTrId("password-needs-to-be-6-characters-or-more");
```

Comment:
Is this still needed with the `StatusMinLengthValidator`?

Reply:
are you referring to the error messages? if so yes, if there are less than 6 characters a "Password needs to be 6 characters or more" will be shown.

Reply:
I'm a bit confused as to what the point of the `StatusMinLengthValidator` is if it doesn't show an error message when the length is < 6 characters.

Reply:
> I'm a bit confused as to what the point of the StatusMinLengthValidator is if it doesn't show an error message when the length is < 6 characters

The reason validators can't know what an error message should look like is primarily i18n (+ letting users of validators decide what their error message is).
I've laid out the entire API and reasoning here: https://github.com/status-im/StatusQ/pull/304#issue-700087306